### DISCUSSION TITLE
tensorrt test failure

### DISCUSSION THREAD

**Discussion by ashkanaev** [apache/incubator-mxnet#14700] [Q&A]
Hi, everyone!

I'm got a problem with deficiencies between outputs of model in mxnet and tensorRT. When i start this test [https://github.com/KellenSunderland/incubator-mxnet/blob/600bc1f1d7b85ff507ede08715e560722829d837/tests/python/tensorrt/test_resnet18.py](url), These is an error 
`AssertionError: 
Items are not equal:
Error 99598.921875 exceeds tolerance rtol=0.000100, atol=0.000100.  Location of maximum error:(356,), a=10.381552, b=0.038473
 a: array([ 0.36684713,  2.8859494 , -0.544983  , ..., -0.7734007 ,
        2.931011  ,  1.8106198 ], dtype=float32)
 b: array([0.0387775 , 0.04588229, 0.03329143, ..., 0.        , 0.        ,
       0.        ], dtype=float32)`

How can i synchronize models outputs? At the moment huge drop in quality makes impossible using tensorrt model.

Software installed: 
mxnet-cu90          1.4.0.post0    
mxnet-tensorrt-cu90 1.3.0   
cudnn-7.5
TensorRT-5.1.2.2

[a comment by an automated account is not shown]

**Comment by frankfliu** [comment]
@mxnet-label-bot add [question]

**Comment by ashkanaev** [comment]
@frankfliu Thank you for reply! 

But it seems as failed unit test. have your building systems passed it? Or you don't check tensorrt integration?  
Hope for your reply!!
Thanks !!

**Comment by ashkanaev** [comment]
@frankfliu 
How can i look at building log/state. Link from main page is outdated i think.
http://jenkins.mxnet-ci.amazon-ml.com/job/incubator-mxnet/job/master/ 
Thanks!
